packages/EasyCore/src/Doctrine/Dispatchers/DeferredEntityEventDispatcher.php

```diff
@@ -36,9 +41,28 @@ public function __construct(EventDispatcherInterface $eventDispatcher)
         $this->eventDispatcher = $eventDispatcher;
     }
 
+    /**
+     * @param int $transactionNestingLevel
```

Comment:
You have defined these params in the interface. Do we really need to define them twice?

Reply:
These params have been removed along with the method.